engine/apps/api/views/alert_receive_channel.py

```diff
@@ -90,6 +78,22 @@ class AlertReceiveChannelView(
 
     filterset_class = AlertReceiveChannelFilter
 
+    rbac_permissions = {
+        "list": [RBACPermission.Permissions.ALERT_RECEIVE_CHANNELS_READ],
```

Comment:
@Matvey-Kuk in the [design doc](https://docs.google.com/document/d/1-GnTcEOIq-XOnAfeNVDnCFUdUaaTzQs8-Ywy52BtBfA/edit#heading=h.4hkvvmp3q5yk) that you authored, "Alert Receive Channels" was not one of the listed object types, should the `AlertReceiveChannel` endpoints use permissions from a different object type?

Reply:
move this to `Integrations` resource type